message_definitions/v1.0/rosflight.xml

```diff
@@ -44,20 +44,20 @@
             <field type="int16_t" name="value4">Fourth value, interpreted according to mode</field>
         </message>
         <message id="181" name="SMALL_IMU">
-            <field type="int16_t" name="stamp">Time in us since last IMU message</field>
+            <field type="int32_t" name="time_boot_us">Measurement timestamp as microseconds since boot</field>
```

Comment:
What if we made this a `uint32_t`
Also, what if we added a `int16_t` temperature field?